src/app/common/app-state.ts

```diff
@@ -1,3 +1,5 @@
+import {allSupportedViews, SupportedView} from '../../../interfaces/shared-interfaces';
+export {allSupportedViews, SupportedView};
```

Comment:
I like your initiative! 😄 

I don't think export is required, is it? Seems a little odd! 🤔

Reply:
That is required If I don't want change import of this file, in the files where this file is imported 😄 Of course I will change it

Reply:
Oh, I see what you mean! Yeah, I think it's cleaner if everywhere just imports from the same location! 👍

Reply:
Done 😄 ... changed imports to shared-interfaces.ts